### PR TITLE
feat: document DAUCustomizer interface

### PR DESCRIPTION
**Pull request by mcollovati**

### CODE REVIEW

File: articles/flow/configuration/licenses/daily-active-users.adoc [hunk 1]
```diff
@@ -12,3 +13,137 @@ Introduction to Daily Active Users ....
...
+[source,java]
+----
+@Singleton
+public class MyDAUCustomizer implements DAUCustomizer {
+
+    @Override
+    public UserIdentitySupplier getUserIdentitySupplier() {
+        return userIdentityContext -> Optional.ofNullable(
+            // In this example a session attribute is supposed to be saved
+            // upon aithentication and then used to provide the user identity
```
Comment (tltv) [inline articles/flow/configuration/licenses/daily-active-users.adoc:105]:
```suggestion
            // upon authentication and then used to provide the user identity
```

File: articles/flow/configuration/licenses/daily-active-users.adoc [hunk 2]
```diff
@@ -12,3 +13,137 @@ Introduction to Daily Active Users ....
...
+            "Please contact customer service.",                  // details
+            "/device-management"                                 // url
+        );
+    }
+}
+----
+
+[NOTE]
+====
+The URL parameter should reference either a static page or a dynanmic page that is not built with Vaadin.
```
Comment (tltv) [inline articles/flow/configuration/licenses/daily-active-users.adoc:146]:
```suggestion
The URL parameter should reference either a static page or a dynamic page that is not built with Vaadin.
```

File: articles/flow/configuration/licenses/daily-active-users.adoc [hunk 3]
```diff
@@ -12,3 +13,137 @@ Introduction to Daily Active Users ....
...
+[source,text]
+.`META-INF/services/com.vaadin.flow.server.dau.DAUCustomizer`
+----
+com.yourpackage.MyDAUCustomizer
+----
+
+
+=== Customizing User Identity Supplier
+
+The user identity supplier is a function that defines how the system identifies a unique user. By default, this feature is not enabled. However, you can provide a custom implementation to count a user only once, regardless of how many devices they use to access your application.
```
Comment (tltv) [inline articles/flow/configuration/licenses/daily-active-users.adoc:91]: Should this be more clear regarding the device. Device means a different browser application, no matter which device it runs on. 
like this: "..., regardless of how many different devices or browser applications on a single device they use to access your application."

Comment (mcollovati) [reply articles/flow/configuration/licenses/daily-active-users.adoc:91]: done

**Review by tltv — APPROVED** [review]

**Pull request merged**